NEWS.md

```diff
@@ -1,3 +1,25 @@
+### 0.12.0
+
+* Revoking deprecation warning of `==` and redefining its meaning as
+  'comparing all fields of two TimeArray'. (#356, #357)
```

Comment:
It would also be good to note here that two `TimeArray`s compare unequal if they have different dimensions.

Reply:
done